RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -735,7 +841,17 @@
                 <Separator />
 
                 <Button Command="{Binding CopyResultsCommand}" CommandParameter="{Binding ElementName=ProjectTree}">
-                    <Image Height="16" Source="../../Resources/document-copy.png" />
+                    <Image Height="16" Source="../../Resources/document-copy.png">
```

Comment:
Yeah scrolling down the menus' markup it's pretty clear to me that these few scarce inline images need to be static resources. Makes maintaining the resources much simpler. I'm not going to hold the PR for that  - or anything at this point, really 💯! Just making a note of the opportunity 😉

Reply:
OK good.  There's more work to be done here; I'll get that another time.  Thx for not holding the PR 😅